examples/readability.js

```diff
@@ -15,7 +15,7 @@
  *   over, loosening constraints each time, if it fails
  * * Happily finds body text in things other than divs and p tags.
  */
-const {readFileSync} = require('fs');
+var fs = require('fs');
```

Comment:
Is this because brfs isn't smart enough to recognize readFileSync if we import it deeply?